### ISSUE TITLE
Metadata: ERR_NOMATCHINGKEYS discussion

### ISSUE THREAD

**Issue by attilamolnar** [ircv3/ircv3-specifications#130]
@DanielOaks commented in #129:

> ERR_NOMATCHINGKEYS is also used in METADATA GET to show that a requested key could not be retrieved, could maybe modify it to ERR_NOMATCHINGKEY or something, but we'd need to leave one of those in for that subcommand unless we rework it.

**Comment by DanielOaks** [comment]
Well, given that `METADATA GET` is the only thing that uses that error, and `METADATA GET` can only return a single key using that numeric, it could make sense to change it to this:
```
| 766 | `ERR_NOMATCHINGKEY`   | `<Key> :no matching key`                 |
```

I'm wondering whether any extensions later on would make use of `ERR_NOMATCHINGKEYS` to refer to multiple keys, similar to the rejected change in #129, but I don't think we need to take that into consideration.

Also semi-related, we should remove `ERR_NOMATCHINGKEYS` from the *Errors* line at the bottom of `METADATA LIST`

**Comment by kaniini** [comment]
I concur with @DanielOaks here

**Comment by DanielOaks** [comment]
This can be closed, superseded by #131
